### DISCUSSION TITLE
Cross references

### DISCUSSION THREAD

**Discussion by horstfortytwo** [knowledgefutures/pubpub#1965] [Support]
I would like to cross ref within the same document. Here is a screenshot of a situation I am talking about:

![image](https://user-images.githubusercontent.com/101699838/164611110-44c5ca5c-fda6-473e-969a-866e0839d8a2.png)

When I use the anchor provided then this anchor is an anchor of the **draft**. When I publish the draft, as far as I can see, the anchor stays an anchor of the **draft**, but that is exactly what I do not want to happen. I would like to cross ref within the same document. When the document is promoted from draft to realease, I would like the link to stay within the release.

I am doind something wrong here? Can I add my anchors explicitly?

**Comment by gabestein** [comment]
Hello, and sorry you're facing this issue. We were just discussing a fix for this; in the meantime, you can simply remove the portion of the URL before the hash and simply use the generated hash itself as the anchor, and it will work across draft and released versions.

**Reply by horstfortytwo** [reply]
Thanks very much. I will do so. - Is there any way to add an anchor explicitly? I tried <a name=...>, but this did not work.
